### DISCUSSION TITLE
Bad ELM response : WRONG RESPONSE : NR: Request Out Of Range(7F2231)

### DISCUSSION THREAD

**Discussion by JeromeTR2909** [cedricp/ddt4all#1177] [Q&A]
Hi everyone,
This shouldn't be a duplicate of the discussion here: https://github.com/cedricp/ddt4all/discussions/1020

I have a 2018 Renault Kadjar and I am trying to set the AUTOMATIC_FOLDING_CF but without luck.
When changing the value from false to true, it gives me this response:
Sending request :DataWrite.AUTOMATIC_FOLDING_CF
Delay 0 ms
Sending ELM request:2E 01 3C 80
ELM response : 6E 01 3C 
Bad ELM response : WRONG RESPONSE : NR: Request Out Of Range(7F2231)
Bad ELM response : WRONG RESPONSE : NR: Request Out Of Range(7F2231)

This is the scanner I have with the right chip in it
https://kingbolentool.com/products/kingbolen-obd2-scanner-bluetooth-check-engine-light?variant=31540052623417

**Comment by AndyIreland** [comment]
Hi, have you got the motors already in the door mirrors that fold them in? - and the switch on the drivers side where you turn the switch to fold the door mirrors in? - and is your touchscreen radio R-link1?

**Reply by JeromeTR2909** [reply]
Hi
Yes the vehicle came with the electric mirrors that fold in. There is a switch where I click it and it folds in and out.
And I believe my R-Link is 2. Anyway I can check that if needed ?

**Reply by AndyIreland** [reply]
so on the r-link2 you might need to make adjustments to that for the folding mirrors - you will need a specially adapted elm327 or OBD2 cable to read write parameters for auto-folding mirrors in rlink2 to make the setting available onscreen

**Reply by AndyIreland** [reply]
even someone said once they could not get auto fold mirrors until they fitted factory fitted original renault alarm module, I dont know if thats true.
